Formula/sambamba.rb

```diff
@@ -0,0 +1,31 @@
+class Sambamba < Formula
+  desc "Tools for working with SAM/BAM data"
+  homepage "https://lomereiter.github.io/sambamba/"
+  url "https://github.com/biod/sambamba/archive/refs/tags/v0.8.2.tar.gz"
+  sha256 "10f7160db5a1c50296b32e94f7d7570739aa7d90c93fe0981246fe89eab6dd98"
+  license "GPL-2.0-or-later"
+
+  depends_on "ldc" => :build
+  depends_on "lz4"
+  uses_from_macos "python" => :build
+  uses_from_macos "zlib"
+
+  resource "testdata" do
```

Comment:
```suggestion
  resource "homebrew-testdata" do
```
So we don't get resource update reminders